ci/before_install.sh

```diff
@@ -91,8 +91,8 @@ msys_install() {
     mingw64/mingw-w64-x86_64-python3
   "
   pacman --noconfirm -S --needed ${packages}
-  botan_pkg="mingw-w64-x86_64-libbotan-2.13.0-1-any.pkg.tar.xz"
-  pacman --noconfirm -U http://repo.msys2.org/mingw/x86_64/${botan_pkg} || \
+  botan_pkg="mingw-w64-x86_64-libbotan-2.16.0-1-any.pkg.tar.zst"
```

Comment:
@rrrooommmaaa Could you please update it here to `mingw-w64-x86_64-libbotan-2.17.3-2-any.pkg.tar.zst` as only since this version MinGW applies Daniel's patch from https://github.com/msys2/MINGW-packages/pull/7640/files? I can do it myself, but then we'll end up with rebase/merge hell due to force-push.

Reply:
P.S. And add some comment why that specific version is used.

Reply:
Yeah sorry I dropped the ball on this but I have to step away for a few days. That is hopefully a quick solution, I was going to just do a makepkg w/thread_utils disabled in CI so we could have 2.16 across the board, but really I don't think it's that important and your solution should be quicker and easier.

Reply:
> @rrrooommmaaa Could you please update it here to `mingw-w64-x86_64-libbotan-2.17.3-2-any.pkg.tar.zst` as only since this version MinGW applies Daniel's patch from https://github.com/msys2/MINGW-packages/pull/7640/files? I can do it myself, but then we'll end up with rebase/merge hell due to force-push.

Sure.